config/src/test/java/com/alibaba/nacos/config/server/utils/GroupKeyTest.java

```diff
@@ -25,43 +28,160 @@
 @WebAppConfiguration
 public class GroupKeyTest {
 
-    @Test
-    public void test_parseGroupKey_非法的() {
-        String key = "11111+222+333333+444";
-        try {
-            GroupKey2.parseKey(key);
-            Assert.fail();
-        } catch (IllegalArgumentException e) {
-            System.out.println(e.toString());
-        }
-
-        key = "11111+";
-        try {
-            GroupKey2.parseKey(key);
-            Assert.fail();
-        } catch (IllegalArgumentException e) {
-            System.out.println(e.toString());
-        }
-
-        key = "11111%29+222";
-        try {
-            GroupKey2.parseKey(key);
-            Assert.fail();
-        } catch (IllegalArgumentException e) {
-            System.out.println(e.toString());
-        }
-
-        key = "11111%2b+222";
-        try {
-            GroupKey2.parseKey(key);
-            Assert.fail();
-        } catch (IllegalArgumentException e) {
-            System.out.println(e.toString());
-        }
-
-        key = "11111%25+222";
-        String[] pair = GroupKey2.parseKey(key);
-        Assert.assertEquals("11111%", pair[0]);
-        Assert.assertEquals("222", pair[1]);
-    }
+  @Rule public final ExpectedException thrown = ExpectedException.none();
+
+  @Test
+  public void getKey_ThreeParams() {
+
+    // Arrange
+    final String dataId = ",";
+    final String group = ",";
+    final String datumStr = "3";
+
+    // Act
+    final String actual = GroupKey.getKey(dataId, group, datumStr);
+
+    // Assert result
+    Assert.assertEquals(",+,+3", actual);
+  }
+
+  @Test
+  public void getKey_TwoParams() {
+
+    // Arrange
+    final String dataId = "3";
+    final String group = "\'";
+
+    // Act
+    final String actual = GroupKey.getKey(dataId, group);
+
+    // Assert result
+    Assert.assertEquals("3+\'", actual);
+  }
+
+  @Test
+  public void getKeyTenant_Plus_ThreeParams() {
+
+    // Arrange
+    final String dataId = "3";
+    final String group = "1";
+    final String tenant = ",";
+
+    // Act
+    final String actual = GroupKey.getKeyTenant(dataId, group, tenant);
```

Comment:
Personally we would declare the above 3 strings inline. But it depends which looks closer to their existing style of tests really.

Reply:
Seems to be a mix, just in-lined them.